docs/guides/migrating-react-router-app.md

```diff
@@ -66,7 +66,7 @@ export default function handleRequest(
   remixContext: EntryContext,
   loadContext: AppLoadContext
 ) {
-  return isbot(request.headers.get("user-agent"))
+  return isbot(request.headers.get("user-agent") || "")
```

Comment:
All docs are updated to reflect the v4 API

Reply:
This use case is supported as is starting isbot@4.2.0

```suggestion
  return isbot(request.headers.get("user-agent"))
```